src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -598,11 +470,11 @@ export default class ReviewConfirmDissolution extends Mixins(DateMixin) {
   background-color: $gray1;
 }
 
-.section-container {
+::v-deep .section-container {
```

Comment:
Is this so your new component inherits this class? 
Was it not without it?

Reply:
yeah, the new component wasn't picking up the section container styling otherwise